log-collector/src/test/scala/io/phdata/pulse/logcollector/SolrCloudStreamTest.scala

```diff
@@ -21,6 +21,8 @@ import io.phdata.pulse.solr.{ BaseSolrCloudTest, TestUtil }
 import org.apache.solr.client.solrj.SolrQuery
 import org.scalatest.FunSuite
 
+import scala.collection.mutable
+
 class SolrCloudStreamTest extends FunSuite with BaseSolrCloudTest {
   val TEST_COLLECTION = TestUtil.randomIdentifier()
```

Comment:
Is this still used?

Reply:
I am pushing the test events in mutable.seq

Reply:
I meant `TEST_COLLECTION`